### ISSUE TITLE
Some past student profiles are displaying twice

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/cdh-web#216]
this is currently visible on the students page with Deborah and Phil's profiles, of which there are two.

# dev notes
this appears directly related to the known bug around logic for queryset annotations with `Greatest()`. the notes already in the code for this method say:
```py
        # show most recent first based on grant or position end date
        # NOTE the use of Case/When here is to avoid Greatest() returning NULL
        # if any of its arguments are NULL, which is mysql behavior:
        # https://docs.djangoproject.com/en/2.2/ref/models/database-functions/#django.db.models.functions.Greatest
        # see also the django docs on conditional aggregation:
        # https://docs.djangoproject.com/en/1.11/ref/models/conditional-expressions/#conditional-aggregation
        # NOTE also that this causes dates to be interpreted as strings in QA;
        # relevant ticket: https://code.djangoproject.com/ticket/30224
```
and indeed, removing the annotation causes the duplicate profiles to disappear. meanwhile, using `.distinct()` does not remove the duplicates as long as the annotation is present.

**Comment by thatbudakguy** [comment]
waiting to see if switching postgres magically resolves this one.

**Comment by thatbudakguy** [comment]
tested and fixed in 3.0 as part of #267.
